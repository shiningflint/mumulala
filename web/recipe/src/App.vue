<template>
  <div class="wrapper">
    <router-view/>
  </div>
</template>

<script>
import Navigation from 'components/layout/Navigation'
export default {
  components: {
    Navigation,
  },
}
</script>

<style lang="scss" scoped>
@import 'stylesheets/vars.scss';

.wrapper {
  background: $light-brown;
  color: $text-color;
  height: 100%;
  line-height: 1.5;
  margin: 0 auto;
  max-width: 600px;
}
</style>
