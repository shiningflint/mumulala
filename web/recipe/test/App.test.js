import { shallowMount, createLocalVue } from '@vue/test-utils'
import App from 'src/App'
import VueRouter from 'vue-router'

const localVue = createLocalVue()
localVue.use(VueRouter)
const router = new VueRouter()

describe('App', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(App, {
      localVue,
      router
    })
  })

  it('Mounts without error', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
