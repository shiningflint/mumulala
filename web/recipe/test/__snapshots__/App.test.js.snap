// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Mounts without error 1`] = `
<div>
  <navigation-stub />
   
  <router-view-stub
    name="default"
  />
</div>
`;
